### ISSUE TITLE
request visualization

### ISSUE THREAD

**Issue by isaryo** [google/next-prediction#22]
I try to visualization of the result by reading #4 and [here](https://github.com/JunweiLiang/social-distancing-prediction/blob/master/code/inference/main.py#L1041) lately.
But I don't understand how to visualization of the result.
Please release that part of the code?
I'm sorry for your inconvenience but thank you for your cooperation.

**Comment by JunweiLiang** [comment]
By using the code in [here](https://github.com/JunweiLiang/social-distancing-prediction/blob/master/code/inference/main.py#L1041), and removing the red circle, you should get exactly the same trajectory visualization as the paper. All you need to do is to add the argmax activity text. What visualization do you get now? What do you expect?

**Comment by isaryo** [comment]
I want to know how to visualize own dataset like [this](https://github.com/JunweiLiang/next-prediction/blob/master/code/prepare_data/README.md).

**Comment by JunweiLiang** [comment]
? Upload the visualization you get according to this [repo](https://github.com/JunweiLiang/social-distancing-prediction) and then tell me what more do you want from that.

**Comment by isaryo** [comment]
Perhaps I think [this](https://drive.google.com/file/d/11G7epf5ETXSEF7Bw0uDztBz7AfiHPMlH/view?usp=sharing) is visualization of the VIRAT_S_000008 result.
I want to visualize of own dataset.
How to visualize other dataset?

**Comment by isaryo** [comment]
Sorry.I could visualize of own dataset.
But, I think long videos is low accuracy to visualize.
Do you have any good solutions in regards to that?

**Comment by JunweiLiang** [comment]
The pre-trained models might only work well on the 45-degree, outdoor, stationery videos. For other videos, you will need to find in-domain data to train new models.
